frontends/web/src/api/exchanges.ts

```diff
@@ -114,7 +114,7 @@ export type TBtcDirectResponse = {
   success: false;
 };
 
-export const getBtcDirectSupported = (code: AccountCode, region: ExchangeRegion['code']) => {
+export const getBtcDirectOTCSupported = (code: AccountCode, region: ExchangeRegion['code']) => {
```

Comment:
Do we expect the OTC service to be available in different regions than the widget? Otherwise we can keep the name and use the endpoint for both

Reply:
I've got this info:

> To get a list of countries you can call the /[api/v1/buy/payment-methods](https://api.btcdirect.eu/api/v1/doc) endpoint to receive a list of payment methods and a list of every country we support, each with the supported payment methods.

> Unfortunately a country pre-select is not possible, the main customizability in the widget is focussed on the order flow rather than the onboarding flow.

Reply:
> Currently our services are available for everyone within the [SEPA](https://en.wikipedia.org/wiki/Single_Euro_Payments_Area) (Single Euro Payments Area). This area consists of all countries in the European Union in addition to Norway, Iceland, Lichtenstein, Switzerland, Monaco, and San Marino.

From https://support.btcdirect.eu/hc/en-gb/articles/360000622189-Which-countries-do-you-offer-services-to